middleware/auth.js

```diff
@@ -24,3 +26,32 @@ module.exports = (req, res, next) => {
         return res.status(HttpStatus.FORBIDDEN).json({ success: false, message: 'No token provided.' });
     }
 }
+
+const AdminAuth = (req, res, next) => {
```

Comment:
I am thinking perhaps a helper function could be made for `Auth` and `AdminAuth` functions since they have basically the same code. Making code more modularized and making changes easier and what not. Perhaps something like: `authHelper(req, res, next, checkFunc)` that will be called from `Auth` and `AdminAuth` just with different parameters: `ValidationUtil.isUserAdmin` if it is `AdminAuth` and nothing if it ordinary `Auth`.